src/editor/FrequencySidebar.js

```diff
@@ -94,38 +74,47 @@ const FrequencySidebar = ( {
 					}
 
 					if ( 'preset_1' === value ) {
```

Comment:
`preset_1` is removed from the options, so this code will never be executed and can be removed.

Reply:
I wanted to keep this code around because we do plan to re-implement presets soon, but I can understand it might cause confusion in the interim. Removed in 65f546.